spid_cie_oidc/provider/tests/test_02_authn_endpoint.py

```diff
@@ -1,12 +1,16 @@
-
 from django.http import HttpRequest
-from django.test import TestCase
+from django.test import Client, TestCase
+from django.urls import reverse
 from spid_cie_oidc.authority.tests.settings import *
 from spid_cie_oidc.entity.jwtse import create_jws
 from spid_cie_oidc.entity.models import FetchedEntityStatement, TrustChain
 from spid_cie_oidc.entity.tests.settings import *
-from spid_cie_oidc.provider.settings import *
+from spid_cie_oidc.entity.utils import (datetime_from_timestamp, exp_from_now,
+                                        iat_now)
+from spid_cie_oidc.provider.tests.authn_endpoint_settings import *
 
+NOW = datetime_from_timestamp(iat_now())
```

Comment:
please create a funciont that returns these values or put these calls where required

in this way these would be assigned during the import and will be unaltered (constant) for ever,runtime